zmq_addon.hpp

```diff
@@ -371,15 +371,15 @@ class active_poller_t
     active_poller_t(active_poller_t &&src) = default;
     active_poller_t &operator=(active_poller_t &&src) = default;
 
-    using handler_t = std::function<void(short)>;
+    using handler_type = std::function<void(event_flags)>;
```

Comment:
Just consistency with in class typedefs of standard library types (value_type, native_handle_type, ...). I could revert it since it is not really relevant to the PR.

Reply:
Hm, yes, that's true. I tried to find some documentation of this convention in the standard library, but there seems to be none. cppzmq has not really committed to follow standard library conventions explicitly for now. But we could do that.

I think it makes sense to change the name, but again it would be preferable to separate this change in a separate commit. Then there is also a dedicated commit message for that explaining the reason for that particular change.